stl/inc/format

```diff
@@ -1116,6 +1116,41 @@ _OutputIt _Write(_OutputIt _Out, monostate) {
     return _Out;
 }
 
+// clang-format off
+template <class _CharT, class _OutputIt, class _Arithmetic>
+    requires (is_arithmetic_v<_Arithmetic> && !same_as<_Arithmetic, bool> && !same_as<_Arithmetic, _CharT>)
+_OutputIt _Write(_OutputIt _Out, _Arithmetic _Value) {
+    // clang-format on
+    // TODO: Reusable buffer
+    array<char, 24> _Buffer;

```

Comment:
Should this magic number be commented? An explicit example for the worst case is `"-1.2345678901234567e-100"`.

Reply:
I think we should have a `inline constexpr size_t _Format_min_buffer_length = 24;` and the appropriate comment for that